packages/massa-web3/src/experimental/basicElements/serializers/arrays.ts

```diff
@@ -159,10 +149,10 @@ export function arrayToBytes(
         args.addI64(value as bigint)
         break
       case ArrayTypes.U32:
-        args.addU32(value as number)
+        args.addU32(value as U32.U32)
```

Comment:
should u32 be a number and not a bigInt?

Reply:
A number in javascript can be a float. 1.1 is a number. U32 is an integer.